Dockerfile.devkit

```diff
@@ -110,6 +110,14 @@ RUN apk add --no-cache \
         --requirement /tmp/requirements-devkit.txt \
     && rm -rf \
         /root/.cache
+RUN mkdir -p /usr/share/ansible/collections \
```

Comment:
Please leave a comment about how you derived this so we can get back to it

Reply:
Good call! Will do. It took a little bit of guesswork and some parsing.

Reply:
I wrote a dev doc.

Reply:
great work